### DISCUSSION TITLE
Cannot add new translations for unauthorized page

### DISCUSSION THREAD

**Discussion by doits** [requarks/wiki#3344] [Help / Questions]
**Describe the bug**

I want to add new translations to test them and followed the [Readme](https://github.com/Requarks/wiki/blob/master/server/locales/README.md). But translations do not show up.

**To Reproduce**
1. Switch base language to `de`
2. Create `server/locales/de.yml`
3. Insert translation
4. Restart server (to be sure)

I tried both versions:

```yml
unauthorized.title: Zugriff verweigert
```

... and ...

```yml
unauthorized:
  title: Zugriff verweigert
```

Server logs shows it is loading them:

```
[...]
Jul 13 18:50:53 hogan node[19472]: 2019-07-13T16:50:53.950Z [MASTER] info: Pulled latest locale updates for German from Graph endpoint: [ COMPLETED ]
Jul 13 18:50:53 hogan node[19472]: 2019-07-13T16:50:53.955Z [MASTER] info: Loaded dev locales from de.yml
Jul 13 18:50:53 hogan node[19472]: 2019-07-13T16:50:53.958Z [MASTER] info: Loaded dev locales from de.yml
Jul 13 18:50:53 hogan node[19472]: 2019-07-13T16:50:53.958Z [MASTER] info: Syncing locales with Graph endpoint: [ COMPLETED ]
[...]
```

**Expected behavior**

Translation shows up.

But still:

<img width="801" alt="Bildschirmfoto 2019-07-13 um 18 56 06" src="https://user-images.githubusercontent.com/148003/61174413-e44ac800-a59f-11e9-93d7-ece2f728053c.png">

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04
 - Wiki.js version: 2.0.0-beta.208
 - Database engine: postgres 10.4

**Comment by doits** [comment]
I managed to translate something in the editor namespace, so maybe the keys are not correct or there is a namespace missing?

**Comment by NGPixel** [comment]
Try setting `offline: true` in your config.yml

Your changes are most likely overwritten by the locales sync.

**Comment by doits** [comment]
Hmm looks like I only cannot translate the unauthorized page, even with `offline: true`.

```yml
   editor:
     backToEditor: Zurück
     assets.fetchImage: Bild laden
     assets.fileCount: 'Anzahl: {{count}}'
     assets.headerActions: Aktion
     assets.headerAdded: Hinzugefügt
```

Those translations work correctly, even with `offline: false`. But the unauthorized page keys seem to not work.

**Comment by doits** [comment]
@NGPixel I changed the title to match more precisely that I only cannot translate the unauthorized page at the moment, probably because I don't know the correct namespace or because it uses a different translation mechanism.

**Comment by Smankusors** [comment]
I think you can help Lokalise project now mentioned in [here](https://docs.requarks.io/dev/translations) instead of creating local translation.
